configuration/pih/pih-config-haiti-cerca.json

```diff
@@ -27,7 +24,8 @@
     "todaysVisits",
     "visitManagement",
     "visitNote",
-    "uhmVitals"
+    "uhmVitals",
```

Comment:
there are two different vitals components, confusingly one named "uhmVitals" and one names "vitals"... not sure which one you want to use, but the differences are examples in this ticket (which is about refactoring it and making it cleaner)

https://pihemr.atlassian.net/browse/UHM-5790

Reply:
For now we are going with uhmVitals